connect-inject/endpoints_controller.go

```diff
@@ -50,6 +50,11 @@ type EndpointsController struct {
 	Log              logr.Logger
 	Scheme           *runtime.Scheme
 	context.Context
+
+	// GetClientFunc allows us to specify how to get a consul client handle.
+	// This is used so that we can provide our own function for testing that is
+	// not dependent on having then ENV set up to pick up tokens and ca certs.
+	GetClientFunc func(string, string, string) (*api.Client, error)
```

Comment:
```suggestion
	GetClient func(string, string, string) (*api.Client, error)
```
I think the "get" verb is enough to imply it's a function, and usually I don't see go function names have `Func` in them.